vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -75,6 +76,22 @@ var (
 	bookPathTemplate = gax.MustCompilePathTemplate("shelves/{shelf}/books/{book}")
 )
 
+func idempotentRetryCodes() []gax.CallOption {
+	return []gax.CallOption{
+		gax.WithRetryCodes([]codes.Code{
+			codes.DeadlineExceeded,
+			codes.Unavailable,
+		}),
```

Comment:
Have you considered the "resource limit exceeded" code? (I forget its exact name.) It could be the result of server-side throttling, so retrying could be one solution (although maybe adaptive throttling is better).